<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrilliantImagery</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20000</width>
      <height>20000</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="ramp_tab">
     <attribute name="title">
      <string>Ramp &amp;&amp; Stabilize</string>
     </attribute>
     <widget class="QLabel" name="ramp_image">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="ramp_opperations_box">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>50</y>
        <width>150</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string>Opperations To Perform</string>
      </property>
      <widget class="QCheckBox" name="ramp_checkbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ramp Linear Properties</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="deflicker_checkbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Deflicker</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="stabilize_checkbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Stabilize</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="data_resue_box">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>150</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Reuse</string>
      </property>
      <widget class="QCheckBox" name="reuse_offsets_checkbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use Previously Calculated Offsets</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="reuse_brightness_checkbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>211</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Use Previously Caluclated Brightnesses</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reload_image_button">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>65</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reload Image</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>260</y>
        <width>227</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="ramp_button">
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notify_completion_checkbox">
         <property name="text">
          <string>Notify Upon Completion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sequence Folder:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ramp_folder_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="ramp_folder_button">
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="export_tab">
     <attribute name="title">
      <string>Video Export</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dng_folder_edit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>DNG Sequence Folder:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Output File:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="rendered_image_folder_edit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Rendered Image Folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="dng_folder_button">
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="output_file_edit"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="output_file_button">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="rendered_image_folder_button">
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sequence Stats:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Image Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="image_count_edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Codec</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QRadioButton" name="prores_ks">
          <property name="text">
           <string>ProRes</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">codec_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="libvpx">
          <property name="text">
           <string>VP9</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">codec_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="libx264">
          <property name="text">
           <string>H.264 (MPEG-4)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">codec_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="libx265">
          <property name="text">
           <string>H.265 (MPEG-H, HEVC)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">codec_buttons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Playback</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QLineEdit" name="frame_rate">
          <property name="text">
           <string>29.97</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Frame Rate (fps)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_1">
          <property name="toolTip">
           <string>Leave blank for default value</string>
          </property>
          <property name="text">
           <string>Bitrate (bits/s)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="bitrate">
          <property name="toolTip">
           <string>Leave blank for default value</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Resolution</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QRadioButton" name="res_1280x720">
          <property name="text">
           <string>720p</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="res_2048x1080">
          <property name="text">
           <string>2K</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="res_3000x2000">
          <property name="text">
           <string>3K</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="res_3840x2160">
          <property name="text">
           <string>4K UHD</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="res_4960x2160">
          <property name="text">
           <string>DCI 4K</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="res_1920x1080">
          <property name="text">
           <string>1080p</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="res_6144x3160">
          <property name="toolTip">
           <string>Not supported by H.264</string>
          </property>
          <property name="text">
           <string>6K</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="res_7680x4320">
          <property name="toolTip">
           <string>Not supported by H.264</string>
          </property>
          <property name="text">
           <string>8K</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="res_source">
          <property name="toolTip">
           <string>Note that not all codecs support all resolutions</string>
          </property>
          <property name="text">
           <string>Source</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">resolution_buttons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>221</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>FFmpeg Inputs</string>
      </property>
      <widget class="QPlainTextEdit" name="ffmpeg_inputs_edit">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>20</y>
         <width>201</width>
         <height>121</height>
        </rect>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="watch_render_button">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Watch &amp;&amp; Render</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Colorspace</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="rgb">
          <property name="text">
           <string>RGB</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">colorspace_buttons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bt709">
          <property name="text">
           <string>BT.709</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">colorspace_buttons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bt2020_cl">
          <property name="text">
           <string>BT.2020</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">colorspace_buttons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Compressioin (for H.264 and VP9)</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="lossless">
          <property name="text">
           <string>Lossless</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>671</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Note: not all attributes are available for all codecs; invalid attributes will be ignored. You may need to do some testing in order to find what works best for you and your needs</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="render_now_button">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Render Now</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="image_render">
     <attribute name="title">
      <string>Image Render</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Imate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="image_render_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="image_render_button">
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="scale_to_fit_checkbox">
         <property name="toolTip">
          <string>This hasn't yet been implimented.</string>
         </property>
         <property name="text">
          <string>Scale to Fit</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="render_scroll_area">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>681</width>
        <height>351</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>679</width>
         <height>349</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_project"/>
    <addaction name="action_save_project"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="action_save_project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="colorspace_buttons"/>
  <buttongroup name="resolution_buttons"/>
  <buttongroup name="codec_buttons"/>
 </buttongroups>
</ui>
